org.contourterminal.App.yml

```diff
@@ -23,7 +23,7 @@ modules:
       - "-DLIBTERMINAL_TESTING=OFF"
     # NOTE: This post-install should be removed with the next release of Contour, when appdata.xml is auto-installed.
     post-install:
-      - install -p -m644 org.contour.Terminal.appdata.xml /app/share/metainfo
+      - install -p -m644 org.contourterminal.App.appdata.xml /app/share/metainfo
```

Comment:
you also ignored my comment about this line.

Reply:
I did not "ignore" it. I was reading it, paused and pondered what exactly you meant. Are you referring to the "right name" argument (which I reasoned about already) or the statement " BTW this also install the file as /app/share/metainfo because the destination is a file name unless -t/--target is used."?
I don't see the problem in that install command as it's going to install the source (the file) into that directory. I was re-reading the man-page's description and I don't see any problem. I am aware of `-t` but I do not see the need for using the 3rd syntax form of install when the 2nd one suffices, too. In a local install call  it all went well. Or am I misunderstanding something here?